src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -779,7 +779,28 @@ public IEnumerable<Element> GetJoinedElements()
                 .ToList();
         }
 
-
+        /// <summary>
+        /// Switch the order of how two elements are geometry is joined. 
+        /// Note that changing this in 3D will also affect your 2D views.
+        /// </summary>
+        /// <param name="cuttingElement">The element that should be cutting the other element</param>
+        /// <param name="otherElement">The other element that are being cut by the cuttingElement</param>
+        /// <returns>Joined elements</returns>
+        public static IEnumerable<Element> SwitchGeometryJoinOrder(Element cuttingElement, Element otherElement)
```

Comment:
this looks like it will `switch` every time it's run, are we handling the case when we want a certain element to be first ? or is that a different node ?

Reply:
The element you input to cuttingElement will always be the first element